.circleci/workflows.yml

```diff
@@ -121,6 +121,8 @@ jobs:
         ferrocene/ci/scripts/llvm_cache.py download
         ./x.py --stage 2 build library src/tools/rustdoc
         ./x.py --stage 1 build src/tools/rustdoc
+        # Build the certified subset of libcore
+        ./x.py build --stage 1 library/core --set rust.std-features='["ferrocene_certified"]'
```

Comment:
This job creates a cache that all future jobs use, so I'm not sure this accomplishes what you want. I suspect that this will either damage or invalidate the cached libcore, possibly causing problems or making the build process longer.

I recommend removing this for now. Once we land custom targets we can add it correctly.